internal/literals/split.go

```diff
@@ -76,6 +76,8 @@ func (x split) obfuscate(data []byte) *ast.BlockStmt {
 		chunks = splitIntoRandomChunks(data)
 	}
 
+	decryptOp := randOperator()
```

Comment:
move this before line 90

Reply:
decide on a single name for the random operator, it is named `decryptOp`, `localKeyOp`, or just `op`